src/sentry/apidocs/examples/issue_alert_examples.py

```diff
@@ -10,3 +10,107 @@ class IssueAlertExamples:
             response_only=True,
         )
     ]
+
+    LIST_PROJECT_RULES = [
+        OpenApiExample(
+            "List issue alert rules for a project",
+            value=[
+                {
+                    "id": "3",
+                    "conditions": [
+                        {
+                            "interval": "1h",
+                            "id": "sentry.rules.conditions.event_frequency.EventFrequencyCondition",
+                            "value": 1000,
+                            "comparisonType": "count",
+                            "name": "The issue is seen more than 1000 times in 1h",
```

Comment:
Can you remove the "name" field from all of these? I did a lot of work recently to make sure it's never saved so it doesn't _hurt_ to add it, but it's not necessary and won't be saved. 

https://github.com/getsentry/sentry/pull/54746 
https://github.com/getsentry/sentry/pull/54915
https://github.com/getsentry/sentry/pull/55091
https://github.com/getsentry/sentry/pull/54739